lib/websocket-server.js

```diff
@@ -275,6 +275,16 @@ class WebSocketServer extends EventEmitter {
 
     cb(ws);
   }
+
+  /**
+   * Provide address of server
+   * @public
+   */
+  address () {
+    if (!this._server) throw new Error(`Server was not initialized`);
+    console.log(this._server.address());
```

Comment:
Replace this line with

```js
if (!this._server) return null;
```

This will handle the case where `address()` is called after `close()`.